contents/blog/how-to-treat-your-pricing-like-a-product.md

```diff
@@ -0,0 +1,102 @@
+
+- Products are akin to living, breathing things that are constantly being changed and improved. New features are added, unused features are (hopefully) removed, and the last commit wasn’t 4 years ago.
+
+- Products are tested – definitely from a code standpoint, and hopefully from a product standpoint. 
+
+- Products have customers who require them to do a certain job. Those customers can be external or internal, and there is some sort of exchange of value for using the product.
+
+It’s easy to see what the products are for nearly any company – at PostHog our analytics suite, session replay, feature flag, and A/B testing tools are all products. They fit the definition above perfectly. 
+
+But pricing is often doesn’t fit this definition. It’s often left static and unchanged for years at a time, it’s not tested from a code or product standpoint, and people don’t even realize that they are customers of this thing because it feels so dead.
+
+This is bad. If pricing is so important to a business that it somehow directly or indirectly determines nearly everything else, we simply cannot be okay with it being dead. 
+
+And to keep it alive, we’ve got to treat it like a product.
+
+But how? I’m glad you asked.
+
+## How to treat your pricing like a product
+
+### 1. Hire or assign a team to own pricing
+A product without an owner is as good as dead. Put someone – or a team of someones – in charge of your pricing and make it a significant portion of their responsibility. If the owner doesn’t have enough bandwidth to work on it, we’re essentially without an owner.
+
+At PostHog, the Growth team (hi! 👋) owns pricing, and I’d estimate it encompasses more than half of our overall responsibility.
+
+### 2. Understand who your customers are and their needs
+Every product has customers, and your pricing product is no different. Who is the customer will vary by business, but it essentially comes down to one question: which people need the pricing (and its tooling) to serve their specific needs?
+
+For PostHog, our pricing product’s customers are:
+
+- **Our users (external)**, who need to clearly see and understand our pricing model so they can decide if they want to sign up and subscribe.
+
+- **Other product teams (internal)**, who need a system that makes it easy and reliable for them to track usage, gate features, subscribe new customers, upsell, etc.
+
+- **Customer success (internal)**, who need a system that allows them to make custom packages for enterprise customers, anticipate invoices, manage accounts, and more.
+
+It’s important to also determine how each customer will transfer value to your product. Since people aren’t paying for pricing as a product (unless, of course, the product you sell is pricing as a product) then the value transfer is murkier. It often comes down to usage, or even revenue on the other products. Whatever it is, understand and track it so you can gauge the usefulness of your efforts.
```

Comment:
```suggestion
It’s also important to determine how each customer will "pay" for or transfer value to your product. Since people aren’t paying for pricing as a product (unless, of course, the product you sell is pricing as a product) then the value transfer is murkier. It often comes down to usage, or even revenue on the other products. Whatever it is, understand and track it so you can gauge the usefulness of your efforts.
```
I get what you mean and ideally I think we would just use "pay" but I see the problem. So maybe use both pay and transfer value?